drivers/clock_control/Kconfig.npcx

```diff
@@ -24,20 +24,32 @@ config CLOCK_NPCX_APB1_PRESCALER
 	range 1 10
 	depends on SOC_FAMILY_NPCX
 	help
-	  APB1 prescaler, allowed values: From 1 to 10.
+	  This sets the APB1 prescaler which changes the APB1_CLK frequency,
+	  allowed values: From 1 to 10. APB1_CLK frequency = OSC_CLK / APB1_PRE.
+	  The frequency must be set to  4MHz <= APB1_CLK <= 50MHz & the APB1_CLK
+	  must be an integer division (including 1) of the frequency of the Core
```

Comment:
What is the meaning of this one? Does it mean that the resulting clock must be a whole number of MHz, or just that this prescaler value is the divisor for OSC_CLK? I assume it is the latter, in which case 'must' doesn't make sense. It just is... so I find this confusing.

Perhaps mention the divisor in the first sentence when you mention prescaler?

Reply:
I adjusted the description. Could you check it again? The prescaler is a divisor for OSC_CLK & the output frequency must follow the requirements.